pallets/dkg-metadata/src/benchmarking.rs

```diff
@@ -28,7 +28,7 @@ use dkg_runtime_primitives::{
 use frame_benchmarking::{benchmarks, impl_benchmark_test_suite};
 use frame_system::RawOrigin;
 use sp_io::crypto::{ecdsa_generate, ecdsa_sign_prehashed, sr25519_generate, sr25519_sign};
-use sp_runtime::{key_types::AURA, traits::TrailingZeroInput, Permill};
+use sp_runtime::{key_types::{AURA, DKG}, traits::TrailingZeroInput, Permill};
```

Comment:
Just in case it got lost in discussion, there is no `key_type::DKG`. The key_type is named `KEY_TYPE` in the `dkg_runtime_primitives` so that is where you'll want to import it.